projects/fmcadc2/Readme.md

```diff
@@ -0,0 +1,8 @@
+# FMCADC2 HDL Project
+
+Here are some pointers to help you:
+  * [Board Product Page](https://www.analog.com/eval-ad-fmcadc2-ebz)
+  * Parts : [12-Bit, 2.6 GSPS/2.5 GSPS/2.0 GSPS, 1.3 V/2.5 V Analog-to-Digital Converter](https://www.analog.com/ad9625)
+  * Project Doc: 
```

Comment:
The project doc should be https://wiki.analog.com/resources/fpga/xilinx/fmc/ad-fmcadc2-ebz